[Bug 231] Fix to filter out empty strings for contract_id (#232)
stellar/stellar-dbt-public: models/docs/marts/hourly_soroban_fee_agg_contract.md

Hourly fee aggregation by contract, sourced from `enriched_history_operations_soroban`. Soroban-only by nature since contract_id only exists on Soroban operations. Provides attribution of fee activity to individual smart contracts, enabling identification of which contracts are driving fee volume or surge pricing.

Strictly per-contract: Soroban operations without a resolvable contract_id are excluded. This covers:
* `invoke_host_function` / `upload_wasm` — no contract exists yet; the uploaded artifact is a `ContractCode` entry identified by its Wasm hash, not a contract address. stellar-etl emits `contract_id = NULL` for these.
* `extend_footprint_ttl` and `restore_footprint` whose footprint touches only `ContractCode` / TTL entries (no `ContractData`) — stellar-etl emits `contract_id = ''` because there is no contract to resolve against. Instances of the same op types whose footprint *does* touch `ContractData` populate `contract_id` normally and are included.

> Because these rows are excluded, totals here do not represent total network-wide Soroban fees; use the network-level Soroban fee aggregation for that.

Author: ```suggestion
Because these rows are excluded, totals here do not represent total network-wide Soroban fees; use the `daily_fee_stats_agg` or `ledger_fee_stats_agg` for full fee agg.
```
